pkg/receive/handler.go

```diff
@@ -109,12 +132,13 @@ type Handler struct {
 	options  *Options
 	listener net.Listener
 
-	mtx          sync.RWMutex
-	hashring     Hashring
-	peers        *peerGroup
-	expBackoff   backoff.Backoff
-	peerStates   map[string]*retryState
-	receiverMode ReceiverMode
+	mtx               sync.RWMutex
+	hashring          Hashring
+	peers             *peerGroup
+	expBackoff        backoff.Backoff
+	peerStates        map[string]*retryState
+	receiverMode      ReceiverMode
+	ActiveSeriesLimit *activeSeriesLimit
```

Comment:
I guess we need only this field exported only because of the meta-monitoring query in `cmd` package, right? How about implementing that method on the `Handler` directly to make it fit with the rest of handler code organization?

Reply:
I did this as I was extracting this logic to another struct! But yes, I could add the `QueryMetamonitoring` method back to `Handler`! 🙂